src/utils/permalinks/Permalinks.js

```diff
@@ -130,6 +130,13 @@ export class RoomPermalinkCreator {
     }
 
     forRoom() {
```

Comment:
This function maps through to `makeRoomPermalink`, which is used for a lot more than the share dialog. We probably want to break this out to a new function named `forSharableRoom()` and use that in `ShareDialog`.